Remora.Discord.Hosting/Remora.Discord.Hosting.csproj

```diff
@@ -5,15 +5,16 @@
     </PropertyGroup>
 
     <PropertyGroup>
-        <Version>5.0.17</Version>
+        <Version>5.0.18</Version>
         <Description>Implementation of a hosted Discord gateway service for the .NET Generic Host</Description>
         <PackageReleaseNotes>
-            Update dependencies.
-        </PackageReleaseNotes>
+			Add options to configure the DiscordService to check slash command support and optionally update slash commands.
+			Update dependencies.
+		</PackageReleaseNotes>
     </PropertyGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Backend\Remora.Discord.Gateway\Remora.Discord.Gateway.csproj" />
+        <ProjectReference Include="..\Remora.Discord.Commands\Remora.Discord.Commands.csproj" />
```

Comment:
Is there a reason this was *replaced*? I question whether this code even compiles.

Reply:
Remora.Discord.Commands includes Gateway and I seen that other remora libs do not reference directly a dependency of a dependency manually so I changed this line only to be Remora.Discord.Commands instead.

Reply:
For them, they could easily implement their own BackgroundService and use that for their own command system. For those who use Remora.Discord.Commands I say an option to add / update commands as well as check for support for slash is a good option in the default DiscordService inside Remora.Discord.Hosting (to make things simple for simple bots).